benchmarks/oss_fuzz_benchmark_integration.py

```diff
@@ -0,0 +1,168 @@
+    """Returns the name of the benchmark. Returns |benchmark_name| if is set.
+    Otherwise returns a name based on |project| and |fuzz_target|."""
+    if benchmark_name is not None:
+        return benchmark_name
+    return project + '_' + fuzz_target
+
+
+def replace_base_builder(benchmark_dir, commit_date):
+    """Replace the parent image of the Dockerfile in |benchmark_dir|,
+    base-builder (latest), with a version of base-builder that is likely to
+    build the project as it was on |commit_date| without issue."""
+    base_builder_repo = bisector._load_base_builder_repo()  # pylint: disable=protected-access
+    if base_builder_repo:
+        base_builder_digest = base_builder_repo.find_digest(commit_date)
+        logs.info('Using base-builder with digest %s.', base_builder_digest)
+        build_specified_commit._replace_base_builder_digest(  # pylint: disable=protected-access
+            os.path.join(benchmark_dir, 'Dockerfile'), base_builder_digest)
+
+
+def create_oss_fuzz_yaml(project, fuzz_target, commit, commit_date, repo_path,
+                         benchmark_dir):
+    """Create the oss-fuzz.yaml file in |benchmark_dir| based on the values from
+    |project|, |fuzz_target|, |commit| and |commit_date|."""
+    yaml_filename = os.path.join(benchmark_dir, 'oss-fuzz.yaml')
+    config = {
+        'project': project,
+        'fuzz_target': fuzz_target,
+        'commit': commit,
+        'commit_date': commit_date,
+        'repo_path': repo_path,
+    }
+    yaml_utils.write(yaml_filename, config)
+
+
+def integrate_benchmark(project,
+                        fuzz_target,
+                        commit,
+                        commit_date,
+                        repo_path,
+                        benchmark_name=None):
+    """Copies files needed to integrate an OSS-Fuzz benchmark and creates the
+    benchmark's oss-fuzz.yaml file."""
+    benchmark_name = get_benchmark_name(project, fuzz_target, benchmark_name)
+    benchmark_dir = os.path.join(benchmark_utils.BENCHMARKS_DIR, benchmark_name)
+    # TODO(metzman): Replace with dateutil since fromisoformat isn't supposed to
+    # work on arbitrary iso format strings. Also figure out if i this timezone
+    # replace correct.
+    commit_date = datetime.datetime.fromisoformat(commit_date).replace(
+        tzinfo=datetime.timezone.utc)
+    copy_oss_fuzz_files(project, commit_date, benchmark_dir)
+    replace_base_builder(benchmark_dir, commit_date)
+    create_oss_fuzz_yaml(project, fuzz_target, commit, commit_date, repo_path,
+                         benchmark_dir)
+
+
+def main():
+    """Copies files needed to integrate an OSS-Fuzz benchmark and creates the
+    benchmark's oss-fuzz.yaml file."""
+    parser = argparse.ArgumentParser(description='Integrate a new benchmark.')
+    parser.add_argument('-p',
+                        '--project',
+                        help='Project for benchmark.',
```

Comment:
consider adding examples to all flag descriptions, so users can understand better what format is expected